src/platform/src/main/java/org/geoserver/platform/resource/Files.java

```diff
@@ -273,20 +273,6 @@ private Files() {
         // utility class do not subclass
     }
 
-    /** @deprecated use {@link Resources#fromURL(Resource, String)} */
```

Comment:
This deprecation may be in error as the method has two "replacements"

* Resources#fromURL( Resource, String ) when used to locate a resource which may be located in a database blob
* Work with files of some sort when locate a data file (for import our datastore access) which must be located on disk

Reply:
I've just applied what the deprecations said, did not design the Resource API. Can you provide a  PR with better alternatives?

Reply:
Yep, if our build comes back. And yeah you are not the first to complain on this point.

Reply:
Here is a PR (against this branch) reverting the one commit, and removing the deprecation on Files.url https://github.com/aaime/geoserver/pull/9